core/src/main/java/org/apache/accumulo/core/conf/cluster/ClusterConfigParser.java

```diff
@@ -61,14 +61,12 @@ private static void flatten(String parentKey, String key, Object value,
         (parentKey == null || parentKey.equals("")) ? "" : parentKey + addTheDot(parentKey);
     if (value instanceof String) {
       results.put(parent + key, (String) value);
-      return;
```

Comment:
This may be easier to follow if the returns were kept, but the else clauses removed. (more of a personnel preference?)
`if(string){
   ...
   return
}
 if(List) {
   ...
   return
}

throw new IllegalStateException`

Reply:
The method is already very short. These aren't needed.

Reply:
I responded too quickly before a meeting I was going to. I'll take another look.